pom.xml

```diff
@@ -422,6 +409,12 @@
       <version>4.13.2</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.awaitility</groupId>
+      <artifactId>awaitility</artifactId>
+      <version>4.0.3</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Cool.

Reply:
Yeah, this library is very practical. We use it a lot in Quarkus. It's a bit harder to get things working properly with wiremock recording though.